static/bidder-info/cointraffic.yaml

```diff
@@ -0,0 +1,10 @@
+endpoint: "https://apps.adsgravity.io/pbs/v1/request"
+maintainer:
+  email: tech@cointraffic.io
```

Comment:
Prebid team has sent email to verify above maintainers email. Requesting to responding back on email thread

Reply:
@sergeykcointraffic Please reply to email sent from us.

Reply:
<img width="795" alt="Screenshot 2024-06-24 at 7 13 46 PM" src="https://github.com/prebid/prebid-server/assets/8147531/d53a6c41-d57e-4cb1-8df3-45981c2f3540">

We have not received it yet. Please send it again. Is the email bouncing?

Reply:
@gargcreation1992 Replied.

Reply:
@stsepelin We have not received it. Could you please check the outbox? Also please check if your email policies allows you to send email outside your org.

Reply:
@gargcreation1992 Everything seems fine. Could you also check the spam folder/filters?

Reply:
I see your responses in the spam folder.

Reply:
I will check if there is anything we can do with this.